src/Behat/Gherkin/Node/FeatureNode.php

```diff
@@ -72,6 +77,7 @@ public function __construct(
         $description,
         array $tags,
         BackgroundNode $background = null,
+        array $rules,
```

Comment:
[BC] we should add this as the last parameter and make it optional (with a default)